Include/internal/pycore_obmalloc.h

```diff
@@ -255,8 +255,8 @@ struct pool_header {
     union { pymem_block *_padding;
             uint count; } ref;          /* number of allocated blocks    */
     pymem_block *freeblock;             /* pool's free list head         */
-    struct pool_header *nextpool;       /* next pool of this size class  */
```

Comment:
Why this removed?

Reply:
In issue-92953
> Pools in the linked list arena.freepools are not guaranteed in the same size class. So comment in the line below may be not accurate.
```
struct pool_header *nextpool;       /* next pool of this size class  */
```
So, I removed it.

Reply:
And it is just a linked list node, which should be consistent with the prev below.